src/app/components/task-progress/task-progress.component.scss

```diff
@@ -54,7 +54,7 @@
     background-color: $progress-task-completed;
 }
 
-@media (max-width: 1248px) and (min-width: 768px) {
+@media (max-width: 1248px) and (min-width: 768px) and (max-width: 480px) {
```

Comment:
just use the same media queries for size 768 either max-width or min-width

Reply:
modified